python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1318,6 +1320,7 @@ def run(
         storage_options: List[str] = [],
         stream: bool = False,
         sysctl: Dict[str, str] = {},
+        systemd: Optional[Union[bool, Literal["always"]]] = None,
```

Comment:
can you add a test case for the boolean options? After that we can merge. I want to make sure it works without problems with `True` and `true` (i wonder if it's case sensitive)

Reply:
The only values accepted by the API (according to the type annotation) are `None`, `True`, `False` and `"always"`, are you saying you think it should accept generic strings? See related issue: https://github.com/gabrieldemarmiesse/python-on-whales/issues/466

Reply:
Ho sorry I wasn't clear, I'm not asking to change the api, I'm asking for more unit tests

Reply:
Ho sorry I wasn't clear, I'm not asking to change the api, I'm asking for more unit tests

Reply:
> I want to make sure it works without problems with `True` and `true` (i wonder if it's case sensitive)

I'm not sure what you meant by this then?

Reply:
In the unit test, we are only testing `systemd="always"`, we should also test `systemd=True` and `systemd=False`

Reply:
> I want to make sure it works without problems with `True` and `true` (i wonder if it's case sensitive)

Did you mean check whether podman accepts `--systemd True`, given the implementation will convert `True` to `"True"` in the command that's run? Since we can only use mock-based unit tests for podman currently and extra test won't actually check this works, but I have checked manually. I can add to the testcase anyway.

Reply:
Indeed it's difficult to have podman working in the CI so mocking will be fine. I just want to make sure the function will keep working if we change the api types

Reply:
I've parameterised the testcase to try all three allowed values - I'm not sure it's really giving much more coverage, but let me know what you think.